README.md

```diff
@@ -77,12 +77,15 @@ You can use Nuxt Storage to get the files from the `<input>` tag:
 
 <script setup>
 	// handleFileInput can handle multiple files
-	const { handleFileInput, files } = useFileStorage()
+	const { handleFileInput, files, filesList } = useFileStorage()
```

Comment:
I took the computer for repair, it may take me two days to work properly. I'm not sure about the performance of these two, let's do a simple test later! But I think sending file objects through 'FormData' is a regular operation

When I first tried to use this plugin, when trying to upload files with suffixes such as ". exe", ". zip", ". tar. gz", and ". apk", it was incorrect to save the extension name in the end

I originally intended to change the 'storeFileLocally' method, but considering that this could be a destructive change, I added a new method based on this

We now have two options: 
1. Keep the 'storeFileLocally' and 'storeFileFormData' methods, allowing users to choose their own usage method. 
2. Keep only one of them, and this method should be able to receive file data from 'FormData'

Do you have any ideas? I am willing to assist

Reply:
I believe the second option is better but we need to test if it will break anything, as far as the developer is concerned the files are being stored and the functions don't need any modifications from the developer. so it will just change the saving mechanism.

The files are being saved no matter how it functions. So ignore efficiency because this is the generally more correct method of saving files but the only thing we should do is check if it will break anything for existing users. 

I'll try to manage time during the next 4 days to test it since your pc is under repair.

I'm not sure if we need to keep the base64 method for any reason but maybe we could export functions that allow the user to utilize some base64 functionality. I'll check for some use cases that might require base64 and determine if it's any useful for the developers

Reply:
Ok! If you don't have time, I can submit a new code in two days and only modify the'storeFileLocally 'method.

Reply:
I appreciate it but we need to test it first for any use case that might break for people who already use the library. 


After checking everything in the library we can completely replace the function with the one you wrote and release it in a 0.3.0 version. We might even go ahead and release it as a stable version soon since the package has no major issues and now uses the standard method of uploading.